cmd/unfork/cli/home.go

```diff
@@ -193,6 +220,32 @@ func (h *Home) handleEvent(e ui.Event) (bool, error) {
 				h.render()
 			}
 		}
+	case "y", "Y":
+		if h.needsOverwritePermission {
+			h.needsOverwritePermission = false
+			// overwrite dir and run unfork
+			localChart := h.localCharts[h.selectedChartIndex-1]
+			upstreamChart := h.upstreamMatches[h.selectedUpstreamIndex-1]
+			unforkPath := path.Join(util.HomeDir(), localChart.HelmName)
+
+			if err := os.RemoveAll(unforkPath); err != nil {
+				panic(err)
+			}
+
+			if err := unforker.Unfork(localChart, upstreamChart); err != nil {
+				panic(err)
+			}
+		}
+	case "n", "N":
+		if h.needsOverwritePermission {
+			h.needsOverwritePermission = false
+			// don't overwrite dir, just run unfork
```

Comment:
i think this will fail, right? i think if the user picks n, it should not run unfork

Reply:
Still WIP - the issue said that this should instead use a dir with a `-n` suffix (which I assume means `-1`, `-2`, etc)

Reply:
And now updated to use suffixes in that situation